src/m_menu.c

```diff
@@ -4188,9 +4187,6 @@ setup_menu_t gen_settings4[] = { // General Settings screen4
   {"Level Brightness", S_THERMO|S_STRICT, m_null, M_X_THRM,
    M_Y + gen4_level_brightness*M_SPC, {"extra_level_brightness"}},
 
-  {"Organize save files", S_YESNO|S_PRGWARN, m_null, M_X,
```

Comment:
But it switches back if no savefiles are found, right? Maybe users want to organize new saves, even if they still have old saves in the port directory (I have a dozen saves that are hard to organize manually).

Reply:
My concern was as follows:
* You don't have savegames in the old location
* The port decides to enable "Organize my savegames"
* You disable it in the menu
* The menu tells you to restart the port for the change to become active
* repeat...

Reply:
We could start with a config value of `-1` and only then check for savegames in the old location and then set the config value to either `0` or `1`.